docs/custom_theme/404.html

```diff
@@ -0,0 +1,18 @@
+{% extends "main.html" %}
+
+<!-- Content block -->
+{% block content %}
+
+<p style="text-align: center;font-size: 3em"><i class="md-icon">account_balance</i></p>
+<h1 id="404-page-not-found" style="text-align: center">404</h1>
+<p style="text-align: center"><strong>Deity not found in the Pantheon</strong></p>
+<p style="text-align: center">
+  Try the <a href="/">homepage</a>, or use the search field on the top of this page.
+</p>
+<p style="text-align: center">
+  If you think we made a mistake and deleted a page that should be here, then please tell us on
+  <a href="{{config.extra.support.gitter}}">{{config.site_name}} Gitter channel</a> or create an issue in
```

Comment:
We're about to move the issues to Jira and remove the ability to create GitHub issues so it might be worth just directing this to the GitHub issues tab (they'll be a redirect on there to Jira).  Otherwise, this will need updating again in a week.